sdk/samples/iot/paho_iot_pnp_sample_common.c

```diff
@@ -748,19 +753,19 @@ static void build_property_payload(
   *out_property_payload = az_json_writer_get_bytes_used_in_destination(&jw);
 }
 
-// build_property_payload_with_status builds a desired JSON status.  This payload is invoked
+// write_property_payload_with_status writes a desired JSON status.  This payload is invoked
 // in response to a desired property, e.g. when the device signals its availability to process
 // a desired temperature request.
-static void build_property_payload_with_status(
+static void write_property_payload_with_status(
     az_span name,
     double value,
-    int32_t ack_code_value,
+    int32_t status_code_value,
     int32_t ack_version_value,
```

Comment:
Should `ack_version_value` also be renamed?

Reply:
Yup will address in coming PR.